react/next/Styled.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components'
```

Comment:
can you avoid including styled-components? it seems to be only used lightly, perhaps this could be done with the style prop

Reply:
Done